templates/CRM/Profile/Form/Dynamic.tpl

```diff
@@ -36,7 +36,7 @@
 
   <div class="crm-submit-buttons">
     <span class="crm-button">{$form._qf_Edit_upload_delete.html}</span>
-    <a class="button cancel" href="{$cancelURL}">{$cancelButtonText}</a>
+    <a class="button cancel" href="{$cancelURL}">{ts}Cancel{/ts}</a>
```

Comment:
Why are you reverting this? This would lose the ability to modify the text in the profile settings. Also, shouldn't this now be conditioned on whether the cancel button option is enabled?

Reply:
The above button seems to only display on a delete form of a multi-record custom field. I didn't see any other places where the above button is rendered. Shouldn't the cancel button on this form only be responsible for closing the popup? But as it already had the href attribute of the cancel url, I've updated the above line to show the cancel text too. 

![image](https://user-images.githubusercontent.com/5929648/38007058-3bcdc4ba-3265-11e8-97c9-6145dcc8a1d7.png)

Reply:
Ahh... I didn't notice that it was limited to deleting. I agree -- in that context, a fixed value of Cancel makes sense.